src/_P064_APDS9960.ino

```diff
@@ -102,25 +199,32 @@ boolean Plugin_064(byte function, struct EventStruct *event, String& string)
         PLUGIN_064_pds = new SparkFun_APDS9960();
 
         String log = F("APDS : ");
-        if ( PLUGIN_064_pds->init() )
+        if (PLUGIN_064_pds->init(PCONFIG(2), PCONFIG(3), PCONFIG(5), PCONFIG(6), PCONFIG(7)) )

```

Comment:
Maybe add a define to replace these `PCONFIG(...)` calls?
For example:
`#define P064_GGAIN  PCONFIG(2)`
Then it is more clear what is used where and also if someone wants to extend the number of parameters, it is clear at the start of the file what PCONFIG fields are already used.

Reply:
I replaced all of them with appropriately named defines.